xdsl/dialects/stencil.py

```diff
@@ -432,6 +479,29 @@ def get(temp: SSAValue | Operation, offset: Sequence[int]):
             result_types=[temp_type.element_type],
         )
 
+    def verify_(self) -> None:
```

Comment:
I would like to add a check that the operand to this op is a block argument of the apply op.

Reply:
Why not, but this should be the job of something like IsolatedFromAbove; a trait ApplyOp have in the OEC implementation.
Because this should be the case of any value accessed by any op in the apply. I'm not sure I want to start it here honestly!

Reply:
oooh okay! Sounds nice!

Reply:
#1122